<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditWithdraw_UI</class>
 <widget class="QDialog" name="creditWithdraw_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOther">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OTHER AMOUNT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>40€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn60">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>60€</string>
   </property>
  </widget>
  <widget class="QLabel" name="depositTitlelLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>CREDIT WITHDRAW</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>20€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>100€</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
